leaders/src/main/java/edu/cornell/TestResultsRecord.java

```diff
@@ -0,0 +1,20 @@
+package edu.cornell;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import org.apache.kafka.common.serialization.Deserializer;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+
+@Data
+@AllArgsConstructor
+public class TestResultsRecord {
```

Comment:
Can converted to a record

```suggestion
public record TestResultsRecord {
```